src/style.css

```diff
@@ -1 +1,102 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+  padding: 0;
+  margin: 0;
+  
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-header {
+  display: none;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 1200px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```

Comment:
remove `position: absolute` and instead of adjust elements with left, top add padding or margin